addons/web/static/src/views/relational_model.js

```diff
@@ -3488,6 +3488,7 @@ export class RelationalModel extends Model {
             rootParams.orderBy = this.defaultOrderBy;
         }
         if (
+            !(rootParams.groupBy && rootParams.groupBy.length) &&
             this.defaultGroupBy &&
             !this.env.inDialog &&
             !(params.groupBy && params.groupBy.length)
```

Comment:
As `rootParams` combines both `this.rootParams` and `params`, do we still have to check params? :thinking: